### PR TITLE
PORT-8540 | Added sendRawDataExamples to ocean docs

### PR DESCRIPTION
**Pull request by mk-armah**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Added `Send Raw Data Examples` flag to all Ocean Integrations

## Updated docs pages
- All Ocean Integrations
- Integrate with Any Tool (`/build-your-software-catalog/sync-data-to-catalog/`)`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/azure/installation.md [hunk 1]
```diff
@@ -418,6 +421,7 @@ docker run -i --rm --platform=linux/amd64 \
   -e OCEAN__PORT__BASE_URL="https://api.getport.io" \
   -e OCEAN__INITIALIZE_PORT_RESOURCES=true \
   -e OCEAN__SEND_RAW_DATA_EXAMPLES=true \
+  -e OCEAN__SEND_RAW_DATA_EXAMPLES=true \
```
Comment (talsabagport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/azure/installation.md:424]: duplicated

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/opsgenie.md [hunk 2]
```diff
@@ -348,6 +352,7 @@ ingest_data:
       docker run -i --rm --platform=linux/amd64 \
         -e OCEAN__EVENT_LISTENER='{"type":"ONCE"}' \
         -e OCEAN__INITIALIZE_PORT_RESOURCES=true \
+        -e OCEAN__SEND_RAW_DATA_EXAMPLES= true  \
```
Comment (talsabagport) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/opsgenie.md:355]:
```suggestion
        -e OCEAN__SEND_RAW_DATA_EXAMPLES=true  \
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/firehydrant.md [hunk 3]
```diff
@@ -293,6 +296,7 @@ steps:
     docker run -i --rm --platform=linux/amd64 \
       -e OCEAN__EVENT_LISTENER='{"type":"ONCE"}' \
       -e OCEAN__INITIALIZE_PORT_RESOURCES=true \
+      -e OCEAN__SEND_RAW_DATA_EXXAMPLES=true  \
```
Comment (talsabagport) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/firehydrant.md:299]:
```suggestion
      -e OCEAN__SEND_RAW_DATA_EXAMPLES=true  \
```

File: docs/build-your-software-catalog/sync-data-to-catalog/project-management/linear/linear.md [hunk 4]
```diff
@@ -358,6 +361,7 @@ ingest_data:
       docker run -i --rm --platform=linux/amd64 \
         -e OCEAN__EVENT_LISTENER='{"type":"ONCE"}' \
         -e OCEAN__INITIALIZE_PORT_RESOURCES=true \
+        -e OCEAN__SEND_RAW_EXAMPLES=true \
```
Comment (talsabagport) [inline docs/build-your-software-catalog/sync-data-to-catalog/project-management/linear/linear.md:364]:
```suggestion
        -e OCEAN__SEND_RAW_DATA_EXAMPLES=true \
```

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md [hunk 5]
```diff
@@ -234,6 +236,7 @@ pipeline {
                             docker run -i --rm --platform=linux/amd64 \
                                 -e OCEAN__EVENT_LISTENER='{"type":"ONCE"}' \
                                 -e OCEAN__INITIALIZE_PORT_RESOURCES=true \
+                                -e OCEAN__SEND_RAW_EXAMPLES=true \
```
Comment (talsabagport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md:239]:
```suggestion
                                -e OCEAN__SEND_RAW_DATA_EXAMPLES=true \
```

**Review by talsabagport — CHANGES_REQUESTED** [review]
Left few comments 👍

**Review by talsabagport — APPROVED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**